sqlgen/reflect_test.go

```diff
@@ -226,37 +212,18 @@ func TestMakeWhere(t *testing.T) {
 	table := s.ByName["users"]
 
 	where, err := makeWhere(table, Filter{"id": 10})
-	if err != nil {
-		t.Error(err)
-	}
-	if !reflect.DeepEqual(where, &SimpleWhere{
-		Columns: []string{"id"},
-		Values:  []interface{}{10},
-	}) {
-		t.Error("bad select")
-	}
+	assert.NoError(t, err)
```

Comment:
did the tests from here down in this file change due to api changes, or as cleanup? if the latter, it'd be nice to have them separate/noted that way for reviewing.

Reply:
😬 it was admittedly cleanup. I tried to extract these all into a separate PR but some of them got away from me